docs/README.md

```diff
@@ -2,7 +2,9 @@
 
 To setup the documentation, first you need to install the dependencies of the full environment. For it please follow the [SETUP.md](../SETUP.md). Then type:
 
+    conda create -n reco_full python=3.6 cudatoolkit=10.0 cudnn>=7.6
     conda activate reco_full
+    pip install -e .[gpu,spark,examples]
```

Comment:
It should be `pip install .[all]` to include everything (you don't need the `-e` I think). 
You should rerun it because you must have missed xlearn.